lib/rails_semantic_logger/extensions/action_dispatch/debug_exceptions.rb

```diff
@@ -0,0 +1,16 @@
+# Log actual exceptions, not a string representation
```

Comment:
do you mind keeping this out of this pr please, as it's in the other one?

Reply:
Yeah sure tksss @cwrw 👍 😄